src/main/kotlin/io/sip3/salto/ce/RoutesCE.kt

```diff
@@ -35,6 +35,8 @@ interface RoutesCE : Routes {
 
     // SDP
     val sdp_session get() = "sdp_session"
```

Comment:
I don't like this `sdp_session_update` and `sdp_session_terminate`. Doesn't match other RoutesCE names. Can we maybe have `sdp_session` as a route and Pair(`update/terminate/send`, payload) as a message?